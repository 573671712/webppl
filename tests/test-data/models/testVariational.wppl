Variational(
    function() {
      var x = flip(0.5);
      var y = flip(0.5);
      factor((x || y) ? 0 : -1);
      return x + y;
    },
    100, 50, 1000, false, 0.1, 0.05);
