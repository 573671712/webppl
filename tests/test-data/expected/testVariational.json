{
  "expectedHist": {
    "0": 0.16,
    "1": 0.48,
    "2": 0.36
  },
  "tolerance": 0.1
}
